lib/ws_servers/api/handlers/on_algo_pause_info_request.js

```diff
@@ -5,6 +5,17 @@ const send = require('../../../util/ws/send')
 
 const { _default: { showAlgoPauseInfo: defaultShowAlgoPauseInfo } } = require('bfx-hf-ui-config').UserSettings
 
+const ALGO_PAUSE_INFO = [
+  'Closing the application while algo orders are active will require you to resume/cancel',
+  'the algo orders when you relaunch the Honey Framework application. \n\nWhen you restart the',
+  'Honey Framework application, you will be prompted to select between the currently active algo orders',
+  'to which two of the given options can be applied:\n\n1. Resume: The atomic orders placed by the',
+  'selected algo orders (if active) will be cancelled, and the algo orders will resume placing orders',
+  'for the remaining amounts. The unselected algo orders will not be resumed and will be lost in perpetuity.',
+  '\n2. Cancel All: All atomic orders placed by the algo orders(if active) will be cancelled and no algo orders',
```

Comment:
orders(if active) missing space

Reply:
will directly use it in UI.. will forward this to UI dev